src/stratis/errors.rs

```diff
@@ -82,3 +87,10 @@ impl From<EngineError> for StratisError {
         StratisError::Engine(err)
     }
 }
+
+/// Allow ability to convert from libudev error to stratis error
```

Comment:
Might as well skip the comment, as it is a very common idiom and none of the others are commented.